training-infra/templates/elasticsearch-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
[NIT] I'd personally prefer to have this in the training-infra/templates/elasticsearch.yaml, but this is also fine, of course